app/pages/reporter/application/[applicationId]/version/[versionNumber]/view.tsx

```diff
@@ -84,9 +90,17 @@ class ViewApplication extends Component<Props> {
     const {session} = this.props.query;
     const {query, router} = this.props;
     const {application} = query;
-    const reviewComments = application?.reviewCommentsByApplicationId.edges.map(
-      (result) => result.node.description
-    );
+    const reviewSteps =
+      application?.applicationReviewStepsByApplicationId.edges;
+    // Merge review comments from all applicationReviewSteps into one list:
+    const reviewComments = reviewSteps?.reduce((mergedStepComments, step) => {
+      const stepComments =
+        step.node.reviewCommentsByApplicationReviewStepId.edges;
+      return [
+        ...mergedStepComments,
+        ...stepComments.map((step) => step.node.description)
+      ];
+    }, []);
```

Comment:
This should probably be abstracted away since it's the same as in `ApplicationWizard.tsx:47`

Reply:
Yes it occurred to me... often I will let myself copy/paste something ONCE and then abstract once I need it a third time. 
Not sure where would be appropriate to put that? `lib/`? I see we now also have a `functions/` folder (but not sure what is the difference)?

Reply:
Looks like this needs to be moved down to `ApplicationDecision`, with a fragment on `Application`

Reply:
it would be some sort of polyfill for `array.flatMap()` which has IE compatibility issues from what I understand, I'd say `lib/` is great for that

Reply:
> Looks like this needs to be moved down to ApplicationDecision, with a fragment on Application

Was kind of trying to avoid that, as I felt that surely we could merge some arrays without needing to create a new fragment container? The ApplicationDecision doesn't want any of the original data about a `ReviewComment` - all it wants is an array of strings to put in a bulleted list.